### PR TITLE
analog update

### PR DESCRIPTION
**Pull request by Pnom**

### CODE REVIEW

File: src/analog/commands/downlink/GetExAbsCurrentMC.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ const examples: TCommandExampleList = [
  *
  * // output command binary in hex representation
  * console.log(command.toHex());
- * // 1f 0f 00
+ * // 1f 0f 0a
```
Comment (DarkPark) [inline src/analog/commands/downlink/GetExAbsCurrentMC.ts:27]: is it correct?

File: src/analog/constants/commandRelations.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+
+// constructor to constructor
+export const requestByResponse = new Map();
+export const responseByRequest = new Map();
+
+// id to constructor
+export const requestById = new Map();
+export const responseById = new Map();
+
```
Comment (DarkPark) [inline src/analog/constants/commandRelations.ts:14]:
```suggestion


```

File: src/analog/commands/uplink/UpdateRunResponse.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+ *
+ * @example
+ * ```js
+ * import UpdateRunResponse from 'jooby-codec/analog/commands/downlink/UpdateRunResponse.js';
+ *
+ * const command = new UpdateRunResponse();
+ *
+ * // output command binary in hex representation
+ * console.log(command.toHex());
+ * // 1f 2c 00
```
Comment (DarkPark) [inline src/analog/commands/uplink/UpdateRunResponse.ts:26]: wrong copypaste from some downlink command

File: src/analog/commands/uplink/VerifyImageResponse.ts [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+ * @example
+ * ```js
+ * import VerifyImageResponse from 'jooby-codec/analog/commands/downlink/VerifyImageResponse.js';
+ *
+ * const parameters = {startTime2000: 756648000, hours: 1, channelList: [1]};
+ * const command = new VerifyImageResponse(parameters);
+ *
+ * // output command binary in hex representation
+ * console.log(command.toHex());
+ * // 1f 2b 01 01
```
Comment (DarkPark) [inline src/analog/commands/uplink/VerifyImageResponse.ts:38]: wrong copypaste from some downlink command

File: src/analog/commands/uplink/VerifyImageResponse.ts [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+ * VerifyImage command parameters
+ */
+interface IVerifyImageParameters {
+    status: boolean,
+}
+
+
+// TODO: rework extended headers detection
+const COMMAND_ID = 0x2b1f;
+
```
Comment (DarkPark) [inline src/analog/commands/uplink/VerifyImageResponse.ts:15]:
```suggestion
```

File: src/analog/commands/downlink/GetExAbsCurrentMC.ts (hunk 1 above)
Comment (Pnom) [reply src/analog/commands/downlink/GetExAbsCurrentMC.ts:27]: there is no `0a` in the example of the command. 
`
const examples: TCommandExampleList = [
    {
        name: 'simple request',
        hex: {header: '1f 0f 00', body: ''}
    }
];
`

File: src/analog/commands/uplink/WriteImageResponse.ts [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+const COMMAND_ID = 0x2a1f;
+const COMMAND_SIZE = 5;
+
+const examples: TCommandExampleList = [
+    {
+        name: 'write image',
+        parameters: {
+            offset: 4,
+            status: true
```
Comment (DeadByElpy) [inline src/analog/commands/uplink/WriteImageResponse.ts:23]: `status` already declared and used as number, like [here](https://github.com/jooby-dev/jooby-codec/blob/main/src/analog/commands/uplink/SetTime2000Response.ts#L13) 

for sameness, it is better to leave the number, or discuss and change it to a Boolean in other commands too

File: src/analog/constants/commandRelations.ts (hunk 2 above)
Comment (Pnom) [reply src/analog/constants/commandRelations.ts:14]: fixed

File: src/analog/commands/uplink/UpdateRunResponse.ts (hunk 3 above)
Comment (Pnom) [reply src/analog/commands/uplink/UpdateRunResponse.ts]: fixed

File: src/analog/commands/uplink/VerifyImageResponse.ts (hunk 4 above)
Comment (Pnom) [reply src/analog/commands/uplink/VerifyImageResponse.ts]: fixed

File: src/analog/commands/uplink/VerifyImageResponse.ts (hunk 5 above)
Comment (Pnom) [reply src/analog/commands/uplink/VerifyImageResponse.ts:15]: fixed

File: src/analog/commands/uplink/WriteImageResponse.ts (hunk 6 above)
Comment (Pnom) [reply src/analog/commands/uplink/WriteImageResponse.ts]: fixed

File: src/analog/commands/downlink/GetExAbsCurrentMC.ts (hunk 1 above)
Comment (Pnom) [reply src/analog/commands/downlink/GetExAbsCurrentMC.ts]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**